bin/harvest_all.rb

```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/env ruby
 
 require 'dotenv/load'
+require 'logger'
 
 Dotenv.load
 
+logger = Logger.new(STDOUT)
 files = Dir["csv/*.csv"]
 files.sort.each do |fn|
   fp = File.expand_path(fn)
-  puts "process #{fp}"
+  logger.info("Processing #{fp}")
```

Comment:
We use puts because we run this from the console and it gives us feedback on progress. If we decide to log it instead, we will have to set up a separate terminal to tail -f the logs.

Reply:
the logger.info command still outputs to the terminal, adding a timestamp and some formatting
![Screenshot 2025-02-25 at 12 17 18 PM](https://github.com/user-attachments/assets/825677f9-a0ca-4eba-a5b6-1251f84c09e8)

Reply:
We will be refactoring this workflow later this month to be an Airflow dag rather than doing the harvesting on the machine like we currently do.